back/engines/commercial/id_vienna_saml/app/lib/id_vienna_saml/citizen_saml_omniauth.rb

```diff
@@ -36,6 +36,11 @@ def profile_to_user_attrs(auth)
       }
     end
 
+    def profile_to_user_attrs_for_update(auth)
+      # Don't update first and last name which are probably generated from email
+      profile_to_user_attrs(auth).slice(:email).transform_values(&:presence).compact
+    end
```

Comment:
I think it would make sense to use `super` here.
```suggestion
    def profile_to_user_attrs_for_update(auth)
      { email: super(auth)[:email].presence }.compact
    end
```